iothub_client/inc/iothub_client_core_common.h

```diff
@@ -149,14 +150,23 @@ extern "C"
 
     MU_DEFINE_ENUM_WITHOUT_INVALID(DEVICE_TWIN_UPDATE_STATE, DEVICE_TWIN_UPDATE_STATE_VALUES);
 
+#define DEVICE_TWIN_MESSAGE_TYPE_VALUES     \
+    DEVICE_TWIN_PATCH_REPORTED_RESPONSE,    \
```

Comment:
Do we really need _PATCH_ and _GET_ in these types?

Reply:
names back to original.  updated to RETRIEVE_PROPERTIES_FULL, RETRIEVE_PROPERTIES_DESIRED, and RETRIEVE_PROPERTIES_REPORTED.